src/app/components/Home/Office/Canvas/PopUpInfoPlace/index.tsx

```diff
@@ -0,0 +1,233 @@
+
+// styles
+import {
+  createStyles,
+  makeStyles,
+  Theme,
+  withStyles,
+  WithStyles,
+} from "@material-ui/core/styles";
+
+import {
+  Button,
+  Dialog,
+  MuiDialogTitle,
+  MuiDialogContent,
+  MuiDialogActions,
+  IconButton,
+  CloseIcon,
+  Typography,
+  TextField,
+} from "./components";
+
+// types
+import { PlaceReservation } from "../../../../../services/BD/type";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      margin: 0,
+      padding: theme.spacing(2),
+    },
+    closeButton: {
+      position: "absolute",
+      right: theme.spacing(1),
+      top: theme.spacing(1),
+      color: theme.palette.grey[500],
+    },
+  });
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    container: {
+      display: "flex",
+      flexWrap: "wrap",
+      flexDirection: "row",
+      justifyContent: "space-between",
+    },
+    textField: {
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
+      width: 250, 
+    },
+    occupantUser: {
+      marginBottom: theme.spacing(2),
+    },
+  })
+);
+export interface DialogTitleProps extends WithStyles<typeof styles> {
+  id: string;
+  children: React.ReactNode;
+  onClose: () => void;
+}
+
+const DialogTitle = withStyles(styles)((props: DialogTitleProps) => {
+  const { children, classes, onClose, ...other } = props;
+  return (
+    <MuiDialogTitle
+      disableTypography
+      className={classes.root}
+      {...other}
+      style={{ textAlign: "center" }}
+    >
+      <Typography variant="h6">{children}</Typography>
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          className={classes.closeButton}
+          onClick={onClose}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </MuiDialogTitle>
+  );
+});
+
+const DialogContent = withStyles((theme: Theme) => ({
+  root: {
+    padding: theme.spacing(3),
+  },
+}))(MuiDialogContent);
+
+const DialogActions = withStyles((theme: Theme) => ({
+  root: {
+    margin: 0,
+    padding: theme.spacing(1),
+  },
+}))(MuiDialogActions);
+
+interface IPopUpInfoPlace {
+  handleEventPopUp: () => void;
+  dataAboutWorkplace: {
+    palceIndex: number;
+    label: string;
+    type: string;
+    occupant: string;
+    startReservation: string;
+    endReservation: string;
+    blocked: boolean;
+  };
+  visibility: boolean;
+}
+
+const PopUpInfoPlace: React.FC<IPopUpInfoPlace> = ({
+  handleEventPopUp,
+  dataAboutWorkplace,
+  visibility,
+}) => {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const user = useSelector(getAuthUser);
+  const selesctedFloor = useSelector(getSelectedFloor);
+  const selectedOffice = useSelector(getSelectedOffice);
+
+  const [startBooking, setStartBooking] = useState("");
+  const [endBooking, setEndBooking] = useState("");
+  const [blockedPlace, setBlockedPlace] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
```

Comment:
better to use complex state with `dataAboutWorkplace` used as initial state value. Later you can use `setState` with function as an argument to merge changes.